sub-packages/bionemo-example_model/README.md

```diff
@@ -1,7 +1,115 @@
-# bionemo-example_model
+## bionemo-example_model
 
-Minimalist package containing an example model that makes use of bionemo2/nemo conventions.
+# Introduction
 
-```bash
-pip install -e .
+This is a minimalist package containing an example model that makes use of bionemo2/nemo conventions. It contains the necessary models, dataloaders, datasets, and custom loss fucntions. The referenced classes and function are in `bionemo.example_model.lightning.lightning_basic`.
+
+This tutorial demonstrates the creation of a simple MNIST model. This should be run in a BioNeMo container. For this tutorial, we will reuse elements from the BioNeMo example_model package.
+
+
+`Megatron`/`NeMo` modules and datasets are special derivatives of PyTorch modules and datasets that extend and accelerate the distributed training and inference capabilities of PyTorch.
+
+Some distinctions of Megatron/NeMo are:
+
+- `torch.nn.Module`/`LightningModule` changes into `MegatronModule`.
+- Loss functions should extend the `MegatronLossReduction` module and implement a `reduce` method for aggregating loss across multiple micro-batches.
+- Megatron configuration classes (for example `megatron.core.transformer.TransformerConfig`) are extended with a `configure_model` method that defines how model weights are initialized and loaded in a way that is compliant with training via NeMo2.
+- Various modifications and extensions to common PyTorch classes, such as adding a `MegatronDataSampler` (and re-sampler such as `PRNGResampleDataset` or `MultiEpochDatasetResampler`) to your `LightningDataModule`.
+
+
+# Loss Functions
+First, we define a simple loss function. These should extend the `MegatronLossReduction` class. The output of forward and backward passes happen in parallel. There should be a forward function that calculates the loss defined. The reduce function is required.
```

Comment:
Could you add a pointer toe the file where this is defined? E.g., "we define a loss function in `xyz.py`"

Reply:
done